### PR TITLE
feat(taplo-lsp): import #:schema-directives relative to file 

### LINKED ISSUE #118: workspace-relative paths only work in the first workspace folder
When I have a vscode project with multiple folders in the workspace, like

```
project1/
  foo.toml
project2/
  bar.toml
```

the lsp mode will set up the `project1` as the workspace directory on initialize:

https://github.com/tamasfe/taplo/blob/5681a56ac51c4b518f1131a43ec2c6fba893bf5c/taplo-lsp/src/handlers.rs#L39-L50

and then never change it.

So if `bar.toml` contains a relative schema header like

```
#:schema ./bar-schema.json
```

the lsp mode will try to resolve it to

```
project1/bar-schema.json
```

instead of 

```
project2/bar-schema.json
```

### PR DESCRIPTION
**Pull request by Profpatsch**
Fixes #118, see commit message.

I also added a bunch of comments while figuring out the logic.

### CODE REVIEW

**Comment by Profpatsch** [comment]
Another improvement that could be made for this schema code: If a schema directive is found, but doesn’t point to a valid schema file, return a lsp message marking the schema directive with the error.

**Review by tamasfe — CHANGES_REQUESTED** [review]

**Comment by tamasfe** [comment]
Thank you, the schema handling logic will require some more thoughts and probably a smaller refactoring (showing errors in diagnostics, and including it in the `taplo-cli` as well), for now this should work just fine.

Other than one thing, this looks good to me.

File: js/vscode/src/extension.ts [hunk 1]
```diff
@@ -46,8 +46,6 @@ export async function activate(context: vscode.ExtensionContext) {
 
   c.registerProposedFeatures();
 
-  output = vscode.window.createOutputChannel("Even Better TOML");
```
[the remark this answers is not in the document]
Comment (tamasfe) [reply js/vscode/src/extension.ts:49]: Ah, I missed the commit message. If this is removed, the global `output` variable should be removed as well not just its lazy initialization.

[the remark this answers is not in the document]
Comment (Profpatsch) [reply js/vscode/src/extension.ts]: I didn’t see that it’s a global variable, I dropped the commit completely. Apparently it’s used in some other parts of the program.

**Comment by Profpatsch** [comment]
Cool, I dropped the commit.

**Pull request merged**